### DISCUSSION TITLE
Compatibilidad Jazztel

### DISCUSSION THREAD

**Discussion by CarlosRodriguez1982** [Paco8/plugin.video.orange.spain#12] [Ideas]
Hola.

Sería posible hacer compatible el plugin con el de Jazztel? Me da la impresion que la aplicacion parece clonada de la de Orange. Por eso no se si habilitando algun ajuste podria hacerse compatible.

Un saludo.

**Comment by Paco8** [comment]
Me comentaron que el addon tal y como está funciona con Jazztel. Pruébalo.

**Comment by CarlosRodriguez1982** [comment]
Ahh jajaj bien bueno es saberlo, yo es que de momento tengo la de Movistar.
